test/bash/linking.bats

```diff
@@ -39,22 +41,22 @@ teardown() {
 
   run link_wrapper git
   assert_success
-  assert [ -f "${USER_HOME}/bin/git" ]
+  assert [ -f "${BIN_DIR}/git" ]
```

Comment:
So git is just a random test executable?

Reply:
yes, I can use a more standard executable like ls, cat or sort like that

Reply:
Leave as is. 🤗